src/types.ts

```diff
@@ -0,0 +1,390 @@
+import Constants from './constants';
+import { parseNumber, valueof } from './utils';
+
+interface IdentitiesByType {
+    [key: number]: string;
+}
+
+export const MessageType = {
+    SessionStart: 1 as const,
+    SessionEnd: 2 as const,
+    PageView: 3 as const,
+    PageEvent: 4 as const,
+    CrashReport: 5 as const,
+    OptOut: 6 as const,
+    AppStateTransition: 10 as const,
+    Profile: 14 as const,
+    Commerce: 16 as const,
+    Media: 20 as const,
+    UserAttributeChange: 17 as const,
+    UserIdentityChange: 18 as const,
+};
+
+export const EventType = {
```

Comment:
From my conversations with @einsteinx2, it sounds like we may be making some architectural changes to our event translation logic. I think rather than making a ticket for sustained engineering, we should only address this for when we decide on what is actually necessary for v3+.